setup.py

```diff
@@ -23,16 +36,20 @@
         'For 2.7 support, please install a version prior to 0.20')
 
 
-dulwich_version_string = '0.20.20'
+DULWICH_VERSION_STRING = '0.20.20'
 
 
 class DulwichDistribution(Distribution):
+    """Dulwich Distribution."""
```

Comment:
Please don't create docstrings that don't say anything useful.

Reply:
Can you tell me what to put in this? Because the establishment of this class is purposeful.

Reply:
A meaningful explanation of what it does and why it exists. If you can just read the class name then the docstring might as well not exist.

Reply:
OK